xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -217,6 +217,8 @@ impl<Network: Get<Option<NetworkId>>, AccountId: From<[u8; 32]> + Into<[u8; 32]>
 {
 	fn convert(location: MultiLocation) -> Result<AccountId, MultiLocation> {
 		let id = match location {
+			// support account IDs from the relay chain
+			MultiLocation { parents: 1, interior: X1(AccountId32 { id, network: None }) } => id,
```

Comment:
Noteworthy change, since this allows interpreting an account on the parent chain into an accountId